<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mpiControlGui</class>
 <widget class="QMainWindow" name="mpiControlGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DetectorMPI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>18</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Input</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_testimages">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>114</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Images</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_linuxpipe">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>114</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Linux Pipe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Data Source</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pipename">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>691</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>/local/madpipe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PipeName</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_imgSizeX">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2048</number>
      </property>
      <property name="singleStep">
       <number>128</number>
      </property>
      <property name="value">
       <number>1024</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_imgSizeY">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>2048</number>
      </property>
      <property name="singleStep">
       <number>128</number>
      </property>
      <property name="value">
       <number>1024</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>X,Y Img Size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_numImgs">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>NumImgs</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_imgPeriodMs">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>90</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Period ms</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Calcs</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_negative">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Negative</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton_tifffile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>114</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Tiff File</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_linuxOutPipe">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>114</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>LinuxPipe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tiffpath">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>611</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>/local</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_tiffbasename">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>testy</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linEdit_tiffnumber">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_linuxOutpipeName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>611</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Path</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>BaseName</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>PipeName</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_nullOutput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>114</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>No Output</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_isPrintTrace">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>97</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Print Trace</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Start</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_start">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string> Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>171</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_numImages">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>400</y>
        <width>341</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_dispImage">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>320</width>
        <height>320</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_dispTextLeft">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_dispTextRight">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>191</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
